nautobot_golden_config/api/serializers.py

```diff
@@ -42,15 +43,11 @@ class Meta:
 class ConfigComplianceSerializer(TaggedObjectSerializer, CustomFieldModelSerializer):
     """Serializer for ConfigCompliance object."""
 
-    url = serializers.HyperlinkedIdentityField(
-        view_name="plugins-api:nautobot_golden_config-api:configcompliance-detail"
-    )
-
     class Meta:
-        """Set Meta Data for ConfigCompliance, will serialize all fields."""
+        """Set Meta Data for ConfigCompliance, will serialize fields."""
 
         model = models.ConfigCompliance
-        fields = "__all__"
+        fields = ("id", "device", "rule", "actual", "intended", "ordered", "compliance")
```

Comment:
Can you explain the updates made here for me? I suspect they are right, but I am not following

Reply:
You're spot on, on this one, I reverted it back.  It was a misunderstanding of how that worked and it doesn't make sense to limit at this time.